### DISCUSSION TITLE
[@types/three] MapControls in wrong file

### DISCUSSION THREAD

**Discussion by RandomNameUser** [DefinitelyTyped/DefinitelyTyped#65259] [Issues with a @types package]
According to types, MapControls is exported by OrbitControls.js (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/three/examples/jsm/controls/OrbitControls.d.ts, bottom). That is not true (any more?), it now has its own file (https://github.com/mrdoob/three.js/blob/dev/examples/jsm/controls/MapControls.js). 

Please update! Thanks!

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
Thanks for the discussion! This is [a change in `r151`](https://github.com/mrdoob/three.js/wiki/Migration-Guide#150--151), which we haven't released types for yet. However there is already [a PR](https://github.com/three-types/three-ts-types/pull/395) to make this change once we get to it. Sorry for the delay on getting the new version released, should have time to work on that later this week.
